src/bagofwords_transformer.jl

```diff
@@ -0,0 +1,87 @@
+"""
+    BagOfWordsTransformer()
+
+    Convert a collection of raw documents to matrix representing a bag-of-words structure.
+
+    Essentially, a bag-of-words approach to representing documents in a matrix is comprised of
+    a count of every word in the document corpus/collection for every document.  This is a simple
+    but often quite powerful way of representing documents as vectors.  The end representation is
+    a matrix with rows representing every document in the corpus and columns representing every word
+    in the corpus.  The value for each cell is the raw count of a particular word in a particular
+    document.
+
+    Similarly to the `TfidfTransformer`, the vocabulary considered can be restricted
+    to words occuring in a maximum or minimum portion of documents.
+"""
```

Comment:
Can you please explicitly mention the kwargs `max_doc_freq` and `min_doc_freq`. You can just tag this onto the end of the last sentence if you want.

Reply:
updated docstring for all three transformers